src/ansys_sphinx_theme/__init__.py

```diff
@@ -442,26 +442,27 @@ def configure_theme_logo(app: Sphinx):
         "image_light": ansys_logo_light_mode,
     }
     theme_options = app.config.html_theme_options
+    logo_option = theme_options.get("logo")
 
-    if not theme_options.get("logo"):
+    if not logo_option:
         theme_options["logo"] = pyansys_logo
 
-    if not isinstance(theme_options.get("logo"), str):
+    if isinstance(logo_option, str) and logo_option not in {"ansys", "pyansys", "no_logo"}:
         raise ValueError(
-            f"Invalid logo option: {theme_options.get('logo')}, The logo option must be either 'ansys', 'pyansys', or 'no_logo'"  # noqa: E501
+            f"Invalid logo option: '{logo_option}', The logo option must be either 'ansys', 'pyansys', or 'no_logo'"  # noqa: E501
         )
 
-    logo = theme_options.get("logo")
+    logo = logo_option
 
     if logo == "ansys":
         theme_options["logo"] = ansys_logo
     elif logo == "pyansys":
         theme_options["logo"] = pyansys_logo
-    elif theme_options.get("logo") == "no_logo":
+    elif logo_option == "no_logo":
```

Comment:
```suggestion
    elif logo == "no_logo":
```

Reply:
resolved in [af05675](https://github.com/ansys/ansys-sphinx-theme/pull/431/commits/af05675afa5d0468c15ea4c3346806ff03affe0c)